container_engine/django_tutorial/README.md

```diff
@@ -0,0 +1,187 @@
+# Getting started with Django on Google Container Engine
+
+This repository is an example of how to run a [Django](https://www.djangoproject.com/) 
+app on Google Managed VMs. It uses the [Writing your first Django app](https://docs.djangoproject.com/en/1
+.9/intro/tutorial01/) Polls application as the example app to deploy. From here on out, we refe to this app as
+the 'polls' application.
+
+## Pre-requisites
+
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com).
+
+2. [Enable billing](https://console.cloud.google.com/project/_/settings) for your project.
+
+3. [Enable APIs](https://console.cloud.google.com/flows/enableapi?apiid=datastore,pubsub,storage_api,logging,plus) for your project. The provided link will enable all necessary APIs, but if you wish to do so manually you will need Datastore, Pub/Sub, Storage, and Logging.
+
+4. Install the [Google Cloud SDK](https://cloud.google.com/sdk)
+
+        $ curl https://sdk.cloud.google.com | bash 
+        $ gcloud init
+
+5. Install [Docker](https://www.docker.com/).
+
+### Create a cluster
+
+Create a cluster for the bookshelf application:
+
+    gcloud container clusters create bookshelf \
+        --scope "https://www.googleapis.com/auth/userinfo.email","cloud-platform" \
+        --num-nodes 2
+    gcloud container clusters get-credentials bookshelf
+
+The scopes specified in the `--scope` argument allows nodes in the cluster to access Google Cloud Platform APIs, such as the Cloud Datastore API.
+
+### Create a Cloud Storage bucket
```

Comment:
I actually changed it so now Django will serve the files during debug. But obviously it's still better to serve from GCS, and we need it for the images in this case anyway.